zitadel/idp_oidc/resource.go

```diff
@@ -0,0 +1,32 @@
+package idp_oidc
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/zitadel/terraform-provider-zitadel/v2/zitadel/helper"
+	"github.com/zitadel/terraform-provider-zitadel/v2/zitadel/idp_utils"
+)
+
+func GetResource() *schema.Resource {
+	return &schema.Resource{
+		Description: "Resource representing an OIDC IDP on the instance.",
+		Schema: map[string]*schema.Schema{
+			helper.OrgIDVar:                helper.OrgIDResourceField,
```

Comment:
idp_oidc does not need an OrgID as it is an instance level resource, please have a look at the other like idp_oauth resource as a reference.

Reply:
This is also the reason the testing fails in the pipeline.